models/user.go

```diff
@@ -457,8 +460,19 @@ func PasswordResetEmail(email string, hostName string, ctx context.Context) bool
 		return false
 	}
 	link := hostName + "/v1/user/password-reset/" + uniq_id + "/" + user.ID.Hex()
-	body := "Please reset your password by clicking on the following link: \n" + link
-	body += "\nThis link will expire in 1 hr"
+	t := template.New("reset_email.html")
+	var err1 error
+	t, err1 = t.ParseFiles("views/reset_email.html")
+	if err1 != nil {
+		log.Println(err1.Error())
+		return false
+	}
+	var tpl bytes.Buffer
+	if err2 := t.Execute(&tpl, map[string]string{"link": link}); err2 != nil {
+		log.Println(err2.Error())
```

Comment:
Also, log sentry errors here.
```go
	hub := sentry.GetHubFromContext(s.Context)
        if hub == nil {
		hub = sentry.CurrentHub()
	}
```

```go
hub.CaptureException(err)
```

Reply:
see this line.
https://github.com/mdg-iitr/codephile-backend/blob/e272663b94f8b5c46b1e0ec456edac33475adf59/scrappers/codeforces/scrapper.go?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L179

Reply:
and this:
https://github.com/mdg-iitr/codephile-backend/blob/e272663b94f8b5c46b1e0ec456edac33475adf59/scrappers/codeforces/scrapper.go?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L203